tests/test_callable.py

```diff
@@ -0,0 +1,141 @@
+import typing
+
+import attr
+import pytest
+
+from attrs_strict import AttributeTypeError, type_validator
+
+try:
+    from collections.abc import Callable
+except ImportError:
+    from collections import Callable
+
+
+@pytest.fixture()
+def simple_callable_type():
+    class UnAnnotatedCallable(Callable):
+        def __call__(self):
+            pass
+
+    return UnAnnotatedCallable
+
+
+@pytest.fixture
+def annotated_callable_int_str_type():
+    class AnnonatedCallable(Callable):
```

Comment:
prefer just functions instead of classes.
Functions by default should be callable. Eg. 

```python
def z(b):
    print(z)
callable(z)
```
`True`